examples/dynamic-listener/dynamic_listener.bal

```diff
@@ -0,0 +1,39 @@
+import ballerina/http;
+import ballerina/lang.runtime;
+
+http:Listener httpListener = new(9090);
+
+var helloService = @http:ServiceConfig {} service {
+
+    resource function sayHello(http:Caller caller,
+        http:Request req) returns error? {
+        // Send a response back to the caller.
+        check caller->respond("Hello, World!");
+    }
+
+    // Resource functions that will shutdown the server
+    resource function shutDownServer(http:Caller caller,
+        http:Request req) returns error? {
+        // Send a response back to the caller.
+        var res1 = caller->respond("Shutting down server");
+        // Stop the listener
+        // This will be automatically called if the program exits by means of a system call
+        var res2 = httpListener.__gracefulStop();
```

Comment:
Instead of `res1`, `res2`, etc. can't we use variable names like `result`, `stopResult`, etc.